src/main/java/com/edu/ComNumber.java

```diff
@@ -0,0 +1,27 @@
+package com.edu;
+
+public class ComNumber {
+
+    public static final int COM_NUMBER_SIZE = 3;
+    public static final int MAX_COM_NUMBER_RANDOM = 9;
+    public static final int COM_NUMBER_START = 1;
+
+    public String comNumCreate() {
+
+        String comNumStr = "";
+        int comNum = 0;
+
+        for (int i = 0; i < COM_NUMBER_SIZE; i++) {
+            comNum = (int) (Math.random() * MAX_COM_NUMBER_RANDOM) +
+                    COM_NUMBER_START;
+            if (!comNumStr.contains(Integer.toString(comNum))) {
+                comNumStr += Integer.toString(comNum);
+            } else {
+                i--;
+            }
```

Comment:
이런 코드 구조는 아래와 같이 if문 하나로 변경할 수 있습니다. 

```suggestion
            if (comNumStr.contains(Integer.toString(comNum))) {
                i--;
                continue;
            }
            
            comNumStr += Integer.toString(comNum);
```
Integer.toString(comNum) 대신에 ``String.valueOf(comNum)`` 을 사용하는게 좋습니다. 

아래의 참고자료를 확인하면 좋을 것 같습니다. 
[Integer.toString() vs String.valueOf()](https://swjeong.tistory.com/146) 

Integer.toString() 의 파라미터가 primitive(=기본형) 라면 Integer.toString()을 써도 무방하지만 Object 타입이라면 null에 대해 ``Null Point Exception``이 발생할 수 있습니다. 

String.valueOf()는 int 타입을 파라미터로 받을 때, 아래와 같이 정의되어있습니다. 
![image](https://user-images.githubusercontent.com/47850258/99892526-05094e80-2cb9-11eb-9f1f-47a6a0e9d8ae.png)

즉 두 함수는 동일합니다. 하지만 Object 타입일 때, valueOf는 아래와 같이 정의되어 null 처리를 하고 있는 것을 알 수 있습니다. 
![image](https://user-images.githubusercontent.com/47850258/99892843-bfe71b80-2cbc-11eb-9990-1ed0cfc45a2c.png)